### ISSUE TITLE
[Feature Request] Add UPDATES and DLC directories

### ISSUE THREAD

**Issue by Neipas09** [Ryujinx/Ryujinx#4776]
### Overview

There's already an option to set the "Game Directory"
![image](https://user-images.githubusercontent.com/58871039/235806491-be414acf-ced6-4b14-86b6-8d7bd0aeb0f9.png)

What i would love is to see an option to set the directories for DLC and Updates as well.
I already have all 3 split up into separate folders thanks to NUT
![image](https://user-images.githubusercontent.com/58871039/235806611-da309ab9-26da-4172-87ad-4649519b485b.png)

With this kind of setup, you could then have Ryujinx auto-add DLC and Updates to the game without any extra steps.

### Smaller details

n/a

### Nature of request

Make ryujinx 1000% better!

### Why would this feature be useful?

No more manual bullshit steps to add Updates and DLC.

**Comment by RandomDude560** [comment]
I agree, updates and DLC should be automatically detected and assigned to their respective game without any user interaction.
But do we really need more directories, why not just use the game directories instead?

**Comment by gm1224** [comment]
> But do we really need more directories, why not just use the game directories instead?

Exactly this. The current path selector will still scan every directory (and subdirectories with the option checked) for any relevant files. The request for automatically applying any updates and DLC has also been up for over a year at https://github.com/Ryujinx/Ryujinx/issues/3265.

**Comment by ashuk2000** [comment]
> > But do we really need more directories, why not just use the game directories instead?
> 
> Exactly this. The current path selector will still scan every directory (and subdirectories with the option checked) for any relevant files. The request for automatically applying any updates and DLC has also been up for over a year at #3265.

A toggle to turn autoloading updates and dlc on or off will be good
